### PR TITLE
Update: netbsd.rawrite32 version 1.0.9.0

### LINKED ISSUE #28552: [netbsd.rawrite32 version 1.0.9.0] Installer hash does not match

### PR DESCRIPTION
**Pull request by OfficialEsco**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

* Resolves #28552

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/28603)

### CODE REVIEW

File: manifests/n/netbsd/rawrite32/1.0.9.0/netbsd.rawrite32.locale.en-US.yaml [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
 PackageIdentifier: netbsd.rawrite32
 PackageVersion: 1.0.9.0
 PackageLocale: en-US
 Publisher: Martin Husemann
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Martin Husemann
 PackageName: Rawrite32
+# PackageUrl:
```
Comment (Trenly) [inline manifests/n/netbsd/rawrite32/1.0.9.0/netbsd.rawrite32.locale.en-US.yaml:13]:
```suggestion
PackageUrl: https://www.netbsd.org/~martin/rawrite32/

```

File: manifests/n/netbsd/rawrite32/1.0.9.0/netbsd.rawrite32.locale.en-US.yaml [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
 
 PackageIdentifier: netbsd.rawrite32
 PackageVersion: 1.0.9.0
 PackageLocale: en-US
 Publisher: Martin Husemann
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl:
```
Comment (Trenly) [inline manifests/n/netbsd/rawrite32/1.0.9.0/netbsd.rawrite32.locale.en-US.yaml:10]:
```suggestion
PublisherUrl: https://www.netbsd.org/
PublisherSupportUrl: https://www.netbsd.org/support
# PrivacyUrl: 
```

**Comment by OfficialEsco** [comment]
`
URI: https://www.netbsd.org/~martin/download/rw32-setup-1.0.9.0.exe, Validation result: Failed, Error Message: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond. 
`

File: manifests/n/netbsd/rawrite32/1.0.9.0/netbsd.rawrite32.locale.en-US.yaml [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+# PublisherUrl: 
+# PublisherSupportUrl: 
+# PrivacyUrl: 
+Author: Martin Husemann
 PackageName: Rawrite32
+# PackageUrl: 
 License: BSD
+# LicenseUrl: 
+# Copyright: 
+# CopyrightUrl:
```
Comment (Trenly) [inline manifests/n/netbsd/rawrite32/1.0.9.0/netbsd.rawrite32.locale.en-US.yaml:17]:
```suggestion
LicenseUrl: https://www.netbsd.org/~martin/rawrite32/license.html
Copyright: © 2000-2018 The NetBSD Foundation, Inc.; © 2000-2020 Martin Husemann; © 2010 Izumi Tsutsui; © 2010 Quentin Garnier; All rights reserved.
CopyrightUrl: https://www.netbsd.org/~martin/rawrite32/license.html
```

**Review by Trenly — COMMENTED** [review]
I'm not sure if winget meets the terms of redistribution based on the license; Someone more knowledgeable than me may need to provide input

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**